orcid-model/src/main/java/org/orcid/jaxb/model/record_rc2/Record.java

```diff
@@ -54,7 +54,7 @@
     protected Person person;
     @XmlElement(namespace = "http://www.orcid.org/ns/activities", name = "activities-summary")
     protected ActivitiesSummary activitiesSummary;
-    @XmlAttribute
+    @XmlTransient
```

Comment:
@amontenegro I was thinking that too, but when I tried, it seemed to work with just @XmlTransient.